src/Sandbox.Server.Http/Startup.cs

```diff
@@ -36,6 +38,14 @@ public void ConfigureServices(IServiceCollection services)
 
             // Business logic dependency injection
             services.AddSingleton<IPersonHandler, PersonHandler>();
+
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new Info { 
```

Comment:
This swagger directive labels the generated documentation with "v1" but doesn't filter the matching API prefix.
See https://github.com/domaindrivendev/Swashbuckle/issues/392